RUFAS/input_manager.py

```diff
@@ -756,259 +731,337 @@ def _add_default_values_to_array_inputs(
 
-    def _array_type_validator(
+    def _validate_array_container_properties(
         self,
+        variable_path: List[str | int],
         variable_properties: Dict[str, Any],
-        var_name: str,
-        input_data_value: List[Any],
+        input_data: Any,
         properties_blob_key: str,
     ) -> bool:
-        """Validates an input data element of type array."""
+        """
+        Validates the container properties of an array input data element.
+
+        Parameters
+        ----------
+        variable_path : List[str | int]
+            The path to the variable being validated.
+        variable_properties : Dict[str, Any]
+            The metadata properties for the variable being validated.
+        input_data : Any
+            The input data to be validated.
+        properties_blob_key : str
+            The metadata properties for the data input file being checked.
+
+        Returns
+        -------
+        bool
+            True if the array container properties are valid, False otherwise.
+        """
+
         info_map = {
             "class": self.__class__.__name__,
-            "function": self._array_type_validator.__name__,
+            "function": self._validate_array_container_properties.__name__,
         }
         properties_violation_message = (
             f"Violates properties defined in metadata properties section" f" '{properties_blob_key}'."
         )
-        if type(input_data_value) is not list:
-            warning_string = "Validation: array is not a list"
-            warning_message = (
-                f"Variable: '{var_name}' is type: {type(input_data_value)}. " f"{properties_violation_message}"
+        variable_path_str = self._convert_variable_path_to_str(variable_path)
+        if not isinstance(input_data, list):
+            om.add_warning(
+                "Validation: array container is not a list",
+                f"Variable: '{variable_path_str}' is not an array but has type: {type(input_data)}. "
+                f"{properties_violation_message}",
+                info_map,
             )
-            om.add_warning(warning_string, warning_message, info_map)
             return False
 
         maximum_length = variable_properties.get("maximum_length")
         minimum_length = variable_properties.get("minimum_length")
         if minimum_length is not None:
-            is_in_range = variable_properties["minimum_length"] <= len(input_data_value)
+            is_in_range = variable_properties["minimum_length"] <= len(input_data)
             if not is_in_range:
-                warning_name = "Validation: array length less than minimum"
-                warning_message = (
-                    f"Variable: '{var_name}' has length: {len(input_data_value)}, less than minimum "
-                    f"length: {minimum_length}. {properties_violation_message}"
+                om.add_warning(
+                    "Validation: array container length less than minimum",
```

Comment:
```suggestion
                    "Validation: array length less than minimum",
```
"array container" makes it sounds like it is something that contains arrays.